app/src/main/java/com/ciandt/dragonfly/example/features/feedback/FeedbackActivity.kt

```diff
@@ -288,72 +282,102 @@ class FeedbackActivity : BaseActivity(), FeedbackContract.View {
             val chips = ArrayList<FeedbackChip>()
             entry.value.mapTo(chips, { FeedbackChip(it) })
 
+            val index = classifications.entries.indexOf(entry)
+
             val classificationsView = ClassificationsView(this)
+            classificationsView.tag = index
             classificationsView.setTitle(entry.key)
             classificationsView.setChips(chips)
-            classificationsView.select(chips.first())
+
+            var select: FeedbackChip = chips.first()
+            chips.forEach {
+                if (negativeClassifications.size == classifications.size && it.classification.title == negativeClassifications[index]) {
```

Comment:
Can we create a variable for this condition, so the intent is clear?

If I got it correctly, it means "is<SOMETHING>Selected"?